### PR TITLE
add controller for API, modify OpenAPI doc

### PR DESCRIPTION
**Pull request by Turall**

### CODE REVIEW

**Comment by ShahriyarR** [comment]
@Turall Thanks for PR.
Could you please explain the purpose of PR in plain English just for convenience? :+1:

**Comment by Turall** [comment]
@ShahriyarR I removed the controller logic from the main.py file. Because main.py is not intended for this. For API routers, I created a controller.py file and registered it in the main.py  file. This approach is more straightforward. I changed the OpenApi schema so that the documentation shows what the endpoints are for.

**Comment by Turall** [comment]
@ShahriyarR additionally added 2 events in the main.py file, when the program starts and when it finishes its work

**Comment by ShahriyarR** [comment]
@Turall Thanks for such great improvements :) 
Merging)
